Series duration of my life.js

```diff
@@ -0,0 +1,42 @@
+const seriesDurations = [
+  {
+    title: "Prison Break",
+    days: 35,
+    hours: 30,
+    minutes: 0,
+  },
+  {
+    title: "Lost",
+    days: 18,
+    hours: 15,
+    minutes: 0,
+  },
+  {
+    title: "24",
+    days: 30,
+    hours: 25,
+    minutes: 0,
+  },
+];
+
+function calPercentageOfLife(seriesDurations){
+ const averageLifespanYears = 80;
```

Comment:
@oyelajakenny This function works very well!
To make it a bit more challenging, let's add support for your own age? 

One option would be to add a second argument `birthYear` and then calculate the number of minutes passed.

Another option (and more advanced) is to support the birth date given as a `Date` object. Here is a hint:

```js
const birthday = new Date("January 15, 1991");
const now = new Date();

// this will give you the number of milliseconds passed since your birth day
// now you just need to convert it to the number of minutes
const millisecondsSince = now.getTime() - birthday.getTime();
```

The example above uses [`getTime()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) function of a Date, and it returns a number of so-called [Unix time](https://en.wikipedia.org/wiki/Unix_time).

You don't have to implement this exercise, as it is a bit more advanced topic, but if you're interested give it a try.